Assets/MixedRealityToolkit/_Core/Inspectors/Utilities/Lines/DataProviders/ParabolaPhysicalLineDataProviderInspector.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Utilities.Lines.DataProviders;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Inspectors.Utilities.Lines.DataProviders
+{
+    [CustomEditor(typeof(ParabolaPhysicalLineDataProvider))]
+    public class ParabolaPhysicalLineDataProviderInspector : BaseMixedRealityLineDataProviderInspector
```

Comment:
Summary Tag

Reply:
Literally none of our other inspectors have summary tags.

Reply:
Except for vNext, they should.